ui/app/templates/components/selectable-card-container-graph.hbs

```diff
@@ -0,0 +1,50 @@
+<SelectableCard
+  @class="grid-item-http"
+  @cardTitle="Http Requests"
+  @total={{totalHttpRequests}}
+  @subText="This month"
+  @gridContainer="true"
```

Comment:
if we already have a property called `gridContainer`, could that be used in the template to conditionally apply the special `grid-item-http` that you're passing in above? that way it's one less property to pass in :D